src/errors.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright 2016 Mozilla
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+use rusqlite;
+
+use mentat_query_parser;
+use mentat_sql;
+
+error_chain! {
```

Comment:
Yeah, ideally we'd like to avoid Err(FooErr(BarErr(BazErr(...)))), but there's no way to do that unless we somehow declare all of our error types top level. And that's what error chain seems to do, so I'm in favor.